provision/core/mariadb/provision.sh

```diff
@@ -98,19 +95,20 @@ function mysql_setup() {
   chmod 0644 "/home/vagrant/.my.cnf"
   vvv_info " * Copied /srv/config/mysql-config/root-my.cnf          to /home/vagrant/.my.cnf"
 
-  check_mysql_root_password
+  if [ "${VVV_DOCKER}" != 1 ]; then
```

Comment:
This whole function would only run if we're not on docker, so i'd say this check is unnecessary.

Reply:
I don't understand, the check prevents it running on docker, this is desirable behaviour